qa-tests-backend/src/test/groovy/Enforcement.groovy

```diff
@@ -680,6 +681,8 @@ class Enforcement extends BaseSpecification {
 
     @Tag("BAT")
     @Tag("PolicyEnforcement")
+    // ROX-18278 Test is flaky since 07/2023. (~2 failures / month)
+    @IgnoreIf({ true })
```

Comment:
Is there an unconditional `@Ignore` perhaps? This looks a bit weird.

Reply:
Good point! 😊